### PR TITLE
Academy page logo update

### PR DESCRIPTION
**Pull request by ashraf-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Comment by ashraf-deriv** [comment]
I am removing academy_logo flag references from all referencing pages since it is not using now.

**Review by salarhfz-fs — APPROVED** [review]

**Review by prince-deriv — APPROVED** [review]

**Comment by sean-binary** [comment]
I believe we can just use the `Nav` instead of `NavAcademy` in layout, since there isn't any differences now. @ashraf-deriv

**Comment by ashraf-deriv** [comment]
> I believe we can just use the `Nav` instead of `NavAcademy` in layout, since there isn't any differences now. @ashraf-deriv

Thanks Sean for the comment, Looks like It is possible, If I am not wrong we still have to handle the visibility of nav bar while scrolling the page specifically for academy page.

**Comment by sean-binary** [comment]
Alright let's keep it separate for separation of concerns, as I overlooked that you would also need to handle `SearchBanner` as well.

**Comment by ashraf-deriv** [comment]
> Alright let's keep it separate for separation of concerns, as I overlooked that you would also need to handle `SearchBanner` as well.

Yes that is right, we can keep it like that for now. Thanks for the review @sean-binary

File: src/pages/academy/components/_search-banner.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import { useBrowserResize } from 'components/hooks/use-browser-resize'
 import { slugify, isBrowser } from 'common/utility'
 import { DerivStore } from 'store'
 import device from 'themes/device'
-import AcademyLogo from 'images/svg/blog/academy-logo.svg'
+import AcademyLogo from 'images/svg/blog/academy-text.svg'
```
Comment (sean-binary) [inline src/pages/academy/components/_search-banner.tsx:16]: Could you also please remove the original `academy-logo.svg` if it is not in used anymore?

Comment (ashraf-deriv) [reply src/pages/academy/components/_search-banner.tsx:16]: @sean Ok noted, will update that. Thank you

**Review by sean-binary — APPROVED** [review]

**Pull request merged**